xmpsdk/src/XMPUtils.cpp

```diff
@@ -5,7 +5,7 @@
 // NOTICE:	Adobe permits you to use, modify, and distribute this file in accordance with the terms
 // of the Adobe license agreement accompanying it.
 // =================================================================================================
-
+#define NOMINMAX
```

Comment:
Huh, why is this suddenly necessary? Also, we shouldn't touch the xmpsdk's source unless absolutely required.

Reply:
@sridharb1 replied _On, windows there is a problem if you don't have it._

Reply:
Then this define should be added via cmake.

Reply:
The CMake code is in fact already including that definition. Take a look at `cmake/compilerFlags.cmake`